pkg/datapath/l2responder/l2responder.go

```diff
@@ -83,136 +82,100 @@ func (p *l2ResponderReconciler) run(ctx context.Context) error {
 	// got out of sync or the map was changed underneath us.
 	ticker := time.NewTicker(5 * time.Minute)
 
+	tbl := p.params.L2AnnouncementTable
+	txn := p.params.StateDB.WriteTxn(tbl)
+	tracker, err := tbl.DeleteTracker(txn, "l2-responder-reconciler")
+	if err != nil {
+		txn.Abort()
+		return fmt.Errorf("delete tracker: %w", err)
+	}
+	txn.Commit()
```

Comment:
Possible follow-up item. This logic looks common enough that could be abstracted in the statedb package.